tests/operators/test_regrid.py

```diff
@@ -152,10 +153,53 @@ def test_regrid_onto_xyspacing_unknown_method(regrid_source_cube):
 
 
 @pytest.mark.filterwarnings("ignore:Selected point is within")
```

Comment:
```suggestion
```
I see you have reduced the boundary margin. This line can thus probably be removed.

Reply:
Line has been removed.